.gitignore

```diff
@@ -11,3 +11,5 @@
 /log/*
 !/log/.keep
 /tmp
+
+/npm-debug.log
```

Comment:
@rrgayhart ,  I added this line to the git ignore file, it is now grey'ed out in my editor but it still shows the npm-debug file down below in this commit.  I don't understand why.  I assumed that adding this to the git ignore, adding, committing then pushing that up to this branch, we would no longer see this file in this commit.  Am I missing something?